protocol/communities/community.go

```diff
@@ -898,6 +901,32 @@ func (o *Community) IsMemberInChat(pk *ecdsa.PublicKey, chatID string) bool {
 	return o.getChatMember(pk, chatID) != nil
 }
 
+// Uses bloom filter members list to estimate presence in the channel.
+// False positive rate is 0.1%.
+func (o *Community) IsMemberLikelyInChat(chatID string) bool {
+	if o.IsControlNode() || o.IsPrivilegedMember(o.MemberIdentity()) || !o.channelEncrypted(chatID) {
+		return true
+	}
+
+	chat, ok := o.config.CommunityDescription.Chats[chatID]
+	if !ok {
+		return false
+	}
+
+	// For communities controlled by clients that haven't updated to newer version yet we assume no membership.
+	if chat.MembersList == nil {
+		return false
+	}
+
+	res, err := testBloomFilter(chat.MembersList, o.config.MemberIdentity, o.ControlNode(), chatID, o.Clock())
```

Comment:
maybe `testBloomFilter` is not the best name? sounds like a test variable :)

Reply:
I wanted to write the same, but thought that it might be a common terminology got bloom filters.
If it's not the case, then better to rename indeed.

Reply:
It is the common terminology but indeed may be misleading in this context. Changed to `verifyMembershipWithBloomFilter`.